src/srr-mapping.ts

```diff
@@ -339,6 +340,18 @@ export function handleCreateCustomHistory(
   );
 }
 
+export function handleAddCustomHistoryToSRR(
+  event: AddCustomHistoryToSRREvent
+): void {
+  logInvocation("handleAddCustomHistoryToSRR", event);
+  let ch = new CustomHistory(event.params.customHistoryId.toString());
+  ch.tokenId = event.params.tokenId.toString();
```

Comment:
I think relation b/w CusomHistory and SRR is Many to Many but is this code ok ? 
or 
`new CustomHistory(event.params.customHistoryId.toString()` is populated at every event emission and co-existed with the same `customHistoryId` ?